metals/src/test/scala/tests/provider/HoverProviderTest.scala

```diff
@@ -1,302 +0,0 @@
-package tests.provider
-
-object HoverProviderTest extends BaseHoverProviderTest {
```

Comment:
I was tempted to keep this test suite since it has many useful tests. I plan to revive it once we have upgraded to the latest Scalameta.